src/components/GlobalPreferences/Network/Network.js

```diff
@@ -0,0 +1,182 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import styled from 'styled-components'
+import {
+  Box,
+  Button,
+  Info,
+  GU,
+  TextInput,
+  textStyle,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import { defaultEthNode, defaultIpfsGateway } from '../../../networks'
+import { setDefaultEthNode, setIpfsGateway } from '../../../local-settings'
+import { InvalidNetworkType, InvalidURI, NoConnection } from '../../../errors'
+import keycodes from '../../../keycodes'
+import {
+  checkValidEthNode,
+  getNetworkType,
+  sanitizeNetworkType,
+} from '../../../lib/web3-utils'
+
+function Network() {
+  const {
+    ethNode,
+    networkType,
+    ipfsGateway,
+    handleEthNodeChange,
+    handleIpfsGatewayChange,
+    handleClearCache,
+    networkError,
+    handleNetworkChange,
+  } = useNetwork()
+  const theme = useTheme()
+
+  const { layoutName } = useLayout()
+  const compact = layoutName === 'small'
+
+  return (
+    <React.Fragment>
+      <Box heading="Node settings">
+        <Label theme={theme}>
+          Ethereum node
+          <TextInput
+            value={ethNode}
+            wide
+            onChange={handleEthNodeChange}
+            css={`
+              ${textStyle('body2')};
+              color: ${theme.contentSecondary};
+            `}
+          />
+          {networkError && (
+            <span
+              css={`
+                ${textStyle('body4')};
+                color: ${theme.negative};
+              `}
+            >
+              {(() => {
+                if (networkError instanceof InvalidNetworkType) {
+                  return `Node must be connected to ${sanitizeNetworkType(
+                    networkType
+                  )}`
+                }
+                if (networkError instanceof InvalidURI) {
+                  return 'Must provide WebSocket endpoint to node'
```

Comment:
```suggestion
                  return 'Must provide Http endpoint to node'
```

As mentioned earlier